_opensearch/query-dsl/term.md

```diff
@@ -264,18 +255,69 @@ GET shakespeare/_search
 ```
 
 You get back documents that match any of the terms.
+#### Sample response
+
+```json
+{
+  "took" : 11,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 2,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "shakespeare",
+        "_id" : "61808",
+        "_score" : 1.0,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 61809,
+          "play_name" : "Merchant of Venice",
+          "speech_number" : 33,
+          "line_number" : "1.3.115",
+          "speaker" : "SHYLOCK",
+          "text_entry" : "Go to, then; you come to me, and you say"
+        }
+      },
+      {
+        "_index" : "shakespeare",
+        "_id" : "61809",
+        "_score" : 1.0,
+        "_source" : {
+          "type" : "line",
+          "line_id" : 61810,
+          "play_name" : "Merchant of Venice",
+          "speech_number" : 33,
+          "line_number" : "1.3.116",
+          "speaker" : "SHYLOCK",
+          "text_entry" : "Shylock, we would have moneys: you say so;"
+        }
+      }
+    ]
+  }
+}
+```
 
-### Terms Lookup
+## Terms lookup query (TLQ)
 
-You can use a `terms` query with a `lookup` to match values based on a field in a document in another index.
+Use a terms lookup query (TLQ) to retrieve multiple field values in a specific document within a specific index. Use the `terms` operation, and specify the index name, document Id and specify the field you want to look up with the `path` parameter.
 
 Parameter | Behavior
 :--- | :---
-`index` | The index from which the document is read.
-`id` | The id of the documented.
-`path` | Path to the field from which the values are used in the terms query.
+`index` | The index name that contains the document that you want search.
```

Comment:
Please change to "The index name that contains the document you want to search." Also, each description should ideally use the same structure (the last two use a different structure than the first).